docs/users/deleting-users.mdx

```diff
@@ -1,5 +1,69 @@
 ---
-sanity_slug: /docs/users/deleting-users
+title: Deleting users
+description: Learn how to delete users from your Clerk backend.
 ---
 
-# This is a stub
+# Deleting users
+
+Clerk currently supports deleting users through our [backend API](https://clerk.com/docs/reference/backend-api/tag/Users#operation/DeleteUser). 
+
+The [`deleteUser()`](/docs/references/backend/user/delete-user) method takes a single argument: the user ID of the user you want to delete. It can be accessed from the `users` sub-api of the `clerkClient`. 
+
+<CodeBlockTabs options={["Curl", "Next.js App Router", "Next.js Pages Router", "Node"]}>
+```bash copy filename="curl.sh"
+curl -XDELETE -H 'Authorization: CLERK_SECRET_KEY' 'https://api.clerk.com/v1/users/{user_id}'
+```
+
+```ts copy filename="app/route/private.ts"
+import { clerkClient } from "@clerk/nextjs";
+import { NextResponse } from 'next/server';
+
+export async function DELETE() {
+  const userId = await body.json().userId;
+
+  try {
+    await clerkClient.users.deleteUser(userId);
+    return NextResponse.json({ message: 'Success' });
+  }
+  catch (error) {
+    console.log(error);
+    return NextResponse.json({ error: 'Error deleting user' });
+  }
+}
+```
```

Comment:
I think this is close, but not quite right. This doc might help: https://nextjs.org/docs/app/building-your-application/routing/route-handlers

Specifically:
* The filepath
* Accessing the request body